### PR TITLE
Add some "e2e" tests on setting and getting attributes

### PR DESCRIPTION
**Pull request by barrucadu**
These tests would have caught #112

### CODE REVIEW

File: spec/requests/e2e_spec.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  context "single update endpoint" do
+    SAMPLE_VALUES.each do |name, value|
+      it "sets '#{name}'" do
+        put "/v1/attributes/test_claim", headers: token_headers, params: { value: value.to_json }
+        expect(response).to be_successful
+
+        expect_attribute_has_value "test_claim", value
+      end
+    end
+  end
```
Comment (AlanGabbianelli) [inline spec/requests/e2e_spec.rb:29]: Should we be using the name too?

```suggestion
  context "single update endpoint" do
    SAMPLE_VALUES.each do |name, value|
      it "sets '#{name}'" do
        put "/v1/attributes/#{name}", headers: token_headers, params: { value: value.to_json }
        expect(response).to be_successful

        expect_attribute_has_value name, value
      end
    end
  end
```

Comment (barrucadu) [reply spec/requests/e2e_spec.rb:29]: No, the `name` here is the name of the test, not the name of the attribute.  I'll update the variable names.

File: spec/requests/e2e_spec.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        it "sets '#{test_name1}' and '#{test_name2}'" do
+          post "/v1/attributes", headers: token_headers, params: { attributes: { test_claim: claim_value1.to_json, test_claim_2: claim_value2.to_json } }
+          expect(response).to be_successful
+
+          expect_attribute_has_value "test_claim", claim_value1
+          expect_attribute_has_value "test_claim_2", claim_value2
+        end
+      end
+    end
+  end
```
Comment (AlanGabbianelli) [inline spec/requests/e2e_spec.rb:43]:
```suggestion
  context "bulk update endpoint" do
    SAMPLE_VALUES.each do |test_name1, claim_value1|
      SAMPLE_VALUES.each do |test_name2, claim_value2|
        it "sets '#{test_name1}' and '#{test_name2}'" do
          post "/v1/attributes", headers: token_headers, params: { attributes: { test_name1 => claim_value1.to_json, test_name2 => claim_value2.to_json } }
          expect(response).to be_successful

          expect_attribute_has_value test_name1, claim_value1
          expect_attribute_has_value test_name2, claim_value2
        end
      end
    end
  end
```

File: spec/requests/e2e_spec.rb (hunk 1 above)
Comment (AlanGabbianelli) [reply spec/requests/e2e_spec.rb]: Ah, I see 😉

**Review by AlanGabbianelli — APPROVED** [review]

**Pull request merged**